tests/fiaas_deploy_daemon/specs/v3/test_v3factory.py

```diff
@@ -439,6 +439,7 @@ def test_additional_labels_and_annotations(self, load_app_config_testdata, facto
             service={"service/label": "true"},
             pod={"pod/label": "true", "s": "override"},
             status={"status/label": "true"},
+            pod_disruption_budget={"pod-disruption-budget/label": "true"},
```

Comment:
Would it make sense to also add a test in the pytest.mark.parametrize annotation?